## Crime Analysis and Clearance Rates in Spain


**This analysis is based on publicly available data from the Spanish Ministry of the Interior. It does not make any legal conclusions or assessments of law enforcement performance. The goal is to explore crime trends and data reporting variations across regions.**

### 1. Business Question  

This project analyzes **crime rates and resolution trends** across Spanish cities.  

#### **Key Questions:**  
- Where are the highest and lowest crime rates?  
- What types of crimes are most common?  
- How do crime resolution rates vary by region?  

#### **Why is this analysis useful?**  
Understanding crime distribution and resolution trends can provide insights for:  
- **Residents & Tourists** – Awareness of crime prevalence in different regions  
- **Local Communities** – Identifying patterns that may impact public safety  
- **Researchers & Policy Analysts** – Examining trends in crime reporting and resolution  

This study offers a **neutral, data-driven perspective** on crime trends and their outcomes. 📊🔍  
 


### 2. Key Metrics Analyzed  

This analysis focuses on key crime statistics and law enforcement performance. The following aspects will be explored:  

1. **Crime Rate by Province**  
   - Top 10 provinces with the highest crime rate per 1,000 residents.  
   - Top 10 provinces with the lowest crime rate per 1,000 residents.  

2. **Crime Resolution by Province**  
   - Percentage of solved crimes across all provinces.  

3. **Crime Distribution by Type**  
   - Breakdown of reported crimes by category.  
   - Breakdown of solved crimes by category.  

4. **Crime Trends Over Time**  
   - Analysis of total reported and solved crimes over the past 10 years.  



### 3. Methodology  

This analysis follows a structured data workflow using multiple tools:  

- **Google Sheets** – Initial data preparation, cleaning, and structuring.  
- **PostgreSQL** – Database storage, SQL queries, and trend analysis.  
- **Power BI & Tableau** – Data visualization and reporting.  
- **R (Jupyter Notebook)** – Data processing, database connection, and additional analysis.  

#### Key Steps:  
1. **Data Preparation** – Cleaning, handling missing values, removing inconsistencies.  
2. **Analysis & Visualization** – SQL queries, statistical insights, and graphical representations.  
3. **General Insights** – Interpretation of findings and key takeaways.  



### 4. Data Acquisition and Crime Classification  

#### Data Overview  
We are working with two datasets obtained from the 
- [Ministerio del Interior de España](https://estadisticasdecriminalidad.ses.mir.es/publico/portalestadistico/)
- [WikipediA](https://es.wikipedia.org/wiki/Anexo:Provincias_y_ciudades_aut%C3%B3nomas_de_Espa%C3%B1a):  

1. **Hechos conocidos** – Number of reported crimes by year, province, and type of offense.  
2. **Hechos esclarecidos** – Number of solved (resolved) crimes by year, province, and type of offense. 
3. **Column 'población'** - Since different provinces have varying population sizes, directly comparing crime counts could introduce biases. To account for this, I collected population data from Wikipedia and added a "population" column. This will allow us to normalize crime data and perform more accurate analyses, considering crime rates per capita rather than absolute values.

#### Crime Categories in Spain

This section provides an overview of the different crime categories reported in Spain. The crimes are classified into 12 main categories, each covering specific types of offenses. The descriptions are provided in both Spanish and English for clarity.
1. **CONTRA LAS PERSONAS** (Crimes Against Persons)
   - Includes offenses such as **homicides, assaults, and domestic violence**.  
   

2. **CONTRA LA LIBERTAD** (Crimes Against Freedom)  
   - Covers **kidnapping, coercion, and threats**.  
   

3. **LIBERTAD SEXUAL** (Crimes Against Sexual Freedom)  
   - Includes **sexual assaults, rape, child exploitation, and pornography**.  
   
   
4. **RELACIONES FAMILIARES** (Crimes Against Family Relations)  
   - Crimes that affect family relationships, including **domestic violence**.
   

5. **PATRIMONIO** (Property Crimes)
   - Covers crimes like **theft, robbery, fraud, vandalism, and money laundering**.  
   

6. **SEGURIDAD COLECTIVA** (Crimes Against Public Safety)
   - Includes **drug trafficking, traffic violations, and other offenses endangering public security**.  
   

7. **FALSEDADES** (Fraud and Forgery)
   - Covers **identity fraud, document forgery, and financial fraud**.  
   

8. **ADMÓN. PÚBLICA** (Crimes Against Public Administration)
   - Includes **bribery, corruption, and offenses against public officials**.  
   

9. **ADMÓN. JUSTICIA** (Crimes Against the Justice System)
   - Crimes affecting the justice system, including **perjury and obstruction of justice**.  
   

10. **ORDEN PÚBLICO** (Crimes Against Public Order)
   - Includes **riots, illegal demonstrations, and unlawful assembly**.  
   

11. **LEGISLACIÓN ESPECIAL** (Special Legislation Violations)
   - Crimes related to **violations of specific regulations and laws, such as environmental offenses**.  
   

12. **OTRAS INFRACCIONES PENALES** (Other Criminal Offenses)
   - Miscellaneous crimes that do not fall into the other categories.  
   


This classification is based on official crime reports in Spain and serves as a reference for understanding the crime data used in the analysis.



***
### 5. Goal  

The primary objective of this analysis is to uncover key insights and trends in crime patterns across Spain.


---
### 6. Data Preparation

Before conducting the analysis, I uploaded two datasets: one containing recorded crimes and another with solved crimes. However, the tables were unstructured and not suitable for analysis.

To make the data readable and analysis-friendly, I performed several preprocessing steps in **Google Sheets**:

- **Checked for missing values (NaN) and duplicates** to ensure data quality.
- **Removed redundant total values**, as they can be calculated later.
- **Eliminated very granular hierarchical data**, since intermediate totals were available. Instead, I used total values for each crime category but distributed them across cities.
- **Deleted the "Desconocida" category**, as all its values were zero, making it irrelevant to the analysis.
- **Removed the "En el extranjero" category**, as it likely refers to crimes committed outside Spain. Since this analysis focuses on crime within Spain, these records were excluded.
- **Trimmed unnecessary spaces** and **corrected data formats** for consistency.
- **Merged** the two datasets into a single structured table for easier analysis.
- **Loaded the cleaned data into a PostgreSQL database** to facilitate calculations and trend analysis.

These preprocessing steps ensured that the dataset was clean, structured, and ready for in-depth exploration.




---
### 7. Data Loading

Before proceeding with the analysis, I loaded the cleaned dataset into the environment.  
This dataset contains crime statistics across Spanish provinces, structured for easier analysis.


In [1]:
library(readr)
library(dplyr)



Attaching package: 'dplyr'


The following objects are masked from 'package:stats':

    filter, lag


The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union




In [2]:
file_path <- "Clean_full_crime.csv"
data_crime <- read_csv(file_path)

Rows: 624 Columns: 23
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr  (2): province, crime_type
dbl (21): poblacion, con_2023, esc_2023, con_2022, esc_2022, con_2021, esc_2...

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.


In [3]:
head(data_crime, 10)

province    poblacion crime_type            con_2023 esc_2023 con_2022
1  Araba/Álava 340420    CONTRA LAS PERSONAS    1386    17        1269   
2  Araba/Álava 340420    CONTRA LA LIBERTAD     1343    16        1121   
3  Araba/Álava 340420    LIBERTAD SEXUAL         146    13         125   
4  Araba/Álava 340420    RELACIONES FAMILIARES    33     1          24   
5  Araba/Álava 340420    PATRIMONIO            11387    81       10282   
6  Araba/Álava 340420    SEGURIDAD COLECTIVA     434     7         368   
7  Araba/Álava 340420    FALSEDADES              331    19         341   
8  Araba/Álava 340420    ADMÓN. PÚBLICA            2     1           4   
9  Araba/Álava 340420    ADMÓN. JUSTICIA         271     5         230   
10 Araba/Álava 340420    ORDEN PÚBLICO           231     5         189   
   esc_2022 con_2021 esc_2021 con_2020 ⋯ con_2018 esc_2018 con_2017 esc_2017
1   10      1027      5        904     ⋯  876      3        806     11      
2   12      1115      9       1071     ⋯ 1173      3       1214     13      
3    5        93      8         95     ⋯  118      5         84      7      
4    3        23      0         16     ⋯   28      2         24      4      
5  134      8032     75       7544     ⋯ 9693     36       9243     67      
6    7       303     14        261     ⋯  309     16        224     14      
7   36       278     29        210     ⋯  168     34        165     25      
8    2         1      0          5     ⋯    3      0          4      0      
9    6       217      4        164     ⋯  192      1        172      3      
10   1       221      4        280     ⋯  144      7        136      8      
   con_2016 esc_2016 con_2015 esc_2015 con_2014 esc_2014
1   795      8       1329      9       1926     13      
2  1169      6        897      6        597      6      
3    49      8         51      4         63      4      
4    26      1         29      1         41      3      
5  9042     85       9372     48       9224     72      
6   215     15        200      5        201      5      
7   149     17        166     31        128     35      
8    11      0         13      0         19      0      
9   194      7        209      4        175      5      
10  116      7        157      5        208      9

In [4]:
str(data_crime)

spc_tbl_ [624 × 23] (S3: spec_tbl_df/tbl_df/tbl/data.frame)
 $ province  : chr [1:624] "Araba/Álava" "Araba/Álava" "Araba/Álava" "Araba/Álava" ...
 $ poblacion : num [1:624] 340420 340420 340420 340420 340420 ...
 $ crime_type: chr [1:624] "CONTRA LAS PERSONAS" "CONTRA LA LIBERTAD" "LIBERTAD SEXUAL" "RELACIONES FAMILIARES" ...
 $ con_2023  : num [1:624] 1386 1343 146 33 11387 ...
 $ esc_2023  : num [1:624] 17 16 13 1 81 7 19 1 5 5 ...
 $ con_2022  : num [1:624] 1269 1121 125 24 10282 ...
 $ esc_2022  : num [1:624] 10 12 5 3 134 7 36 2 6 1 ...
 $ con_2021  : num [1:624] 1027 1115 93 23 8032 ...
 $ esc_2021  : num [1:624] 5 9 8 0 75 14 29 0 4 4 ...
 $ con_2020  : num [1:624] 904 1071 95 16 7544 ...
 $ esc_2020  : num [1:624] 3 9 8 3 52 10 30 0 4 2 ...
 $ con_2019  : num [1:624] 927 1101 82 25 9980 ...
 $ esc_2019  : num [1:624] 9 14 3 3 66 10 27 1 3 1 ...
 $ con_2018  : num [1:624] 876 1173 118 28 9693 ...
 $ esc_2018  : num [1:624] 3 3 5 2 36 16 34 0 1 7 ...
 $ con_2017  : num [1:624] 8

In [6]:
file.edit("~/.Renviron")


In [7]:
library(DBI)
library(RPostgres)

con <- dbConnect(
  RPostgres::Postgres(),
  dbname = "postgres",
  host = "localhost",
  port = 5432,
  user = Sys.getenv("DB_USER"),
  password = Sys.getenv("DB_PASS")
)



In [8]:
dbGetQuery(con, "SELECT * FROM crime LIMIT 5;")


province    poblacion crime_type            con_2023 esc_2023 con_2022
1 Araba/Álava 340420    CONTRA LAS PERSONAS    1386    17        1269   
2 Araba/Álava 340420    CONTRA LA LIBERTAD     1343    16        1121   
3 Araba/Álava 340420    LIBERTAD SEXUAL         146    13         125   
4 Araba/Álava 340420    RELACIONES FAMILIARES    33     1          24   
5 Araba/Álava 340420    PATRIMONIO            11387    81       10282   
  esc_2022 con_2021 esc_2021 con_2020 ⋯ con_2018 esc_2018 con_2017 esc_2017
1  10      1027      5        904     ⋯  876      3        806     11      
2  12      1115      9       1071     ⋯ 1173      3       1214     13      
3   5        93      8         95     ⋯  118      5         84      7      
4   3        23      0         16     ⋯   28      2         24      4      
5 134      8032     75       7544     ⋯ 9693     36       9243     67      
  con_2016 esc_2016 con_2015 esc_2015 con_2014 esc_2014
1  795      8       1329      9       1926     13      
2 1169      6        897      6        597      6      
3   49      8         51      4         63      4      
4   26      1         29      1         41      3      
5 9042     85       9372     48       9224     72

---
### 8. Analysis & Visualization  

In this section, I integrate SQL, Tableau, and Power BI to analyze and visualize crime data comprehensively. By combining these tools, all insights are presented in one structured section, making it easier to explore trends and crime patterns without scattering the analysis across different parts of the project.


---
#### 8.1. Crime Rate per 1,000 People  

In this analysis, we examine which provinces have the highest and lowest crime rates in Spain. To ensure fairness, we calculate crimes per 1,000 residents rather than using absolute numbers, as population sizes vary significantly between provinces. This approach allows for an accurate and standardized comparison across all regions.


In [9]:
dbGetQuery(con, 
           "select  province, sum(con_2023) as total_crime, 
		            sum(esc_2023) as total_solved,
                    round(cast(sum(esc_2023)/ sum(con_2023) * 100.0 as numeric), 1) as percent_crime_solved
            from crime 
            group by province
            order by percent_crime_solved desc;")

province               total_crime total_solved percent_crime_solved
1  Melilla                  4357        2743       63.0                
2  Ceuta                    3541        2021       57.1                
3  Ciudad Real             15698        8256       52.6                
4  Palencia                 4915        2557       52.0                
5  Palmas (Las)            56626       28717       50.7                
6  Cáceres                 11615        5816       50.1                
7  Asturias                33888       16528       48.8                
8  Almería                 33291       16078       48.3                
9  Lugo                     9155        4288       46.8                
10 Santa Cruz de Tenerife  47596       21955       46.1                
11 Granada                 41752       18806       45.0                
12 Zamora                   5477        2430       44.4                
13 Ourense                  9624        4261       44.3                
14 Pontevedra              33910       15002       44.2                
15 Huelva                  23463       10343       44.1                
16 Jaén                    21073        9262       44.0                
17 Cádiz                   59907       25926       43.3                
18 Rioja (La)              12152        5237       43.1                
19 Córdoba                 31810       13651       42.9                
20 Albacete                15618        6675       42.7                
21 Cantabria               24212       10331       42.7                
22 Cuenca                   7394        3138       42.4                
23 Málaga                  99025       41331       41.7                
24 Segovia                  5950        2483       41.7                
25 Soria                    3715        1548       41.7                
26 Badajoz                 23654        9834       41.6                
27 Alicante/Alacant       105373       43747       41.5                
28 Toledo                  32325       13285       41.1                
29 Murcia                  74194       30514       41.1                
30 Huesca                   7987        3264       40.9                
31 León                    16827        6814       40.5                
32 Ávila                    6235        2521       40.4                
33 Salamanca               12695        5095       40.1                
34 Zaragoza                38899       15545       40.0                
35 Castellón/Castelló      29150       11615       39.8                
36 Baleares (Illes)        78509       31005       39.5                
37 Coruña (A)              42111       16640       39.5                
38 Teruel                   3979        1565       39.3                
39 Burgos                  13970        5343       38.2                
40 Valladolid              20040        7570       37.8                
41 Valencia/València      152856       57090       37.3                
42 Guadalajara             12673        4428       34.9                
43 Lleida                  22354        7727       34.6                
44 Sevilla                102860       35366       34.4                
45 Navarra                 35094       11264       32.1                
46 Madrid                 411782      127310       30.9                
47 Tarragona               46897       14157       30.2                
48 Girona                  48223       14275       29.6                
49 Barcelona              392698       88457       22.5                
50 Araba/Álava             15676         176        1.1                
51 Bizkaia                 59227         485        0.8                
52 Gipuzkoa                32920         257        0.8

![Tableau Visualization](images/crime10bad.png)


![Tableau Visualization](images/screen10min.png)




From the visualizations, we observe a clear trend:  

- **Higher crime rates** are concentrated in larger urban areas such as Barcelona, Madrid, and the Balearic Islands.  
  - These locations attract high numbers of tourists, which may contribute to higher crime rates due to factors like pickpocketing, fraud, and petty theft.  
  - Additionally, higher population density and urban dynamics naturally lead to more reported crimes.  

- **Lower crime rates** are found in smaller provinces like Lugo, Teruel, and Cáceres.  
  - These areas have lower population density, less tourism, and generally less economic activity, which may contribute to fewer reported crimes.  
  - Rural and less populated regions often experience less violent or organized crime, focusing more on minor infractions.  

This data highlights the impact of urbanization and tourism on crime rates, emphasizing that crime trends are not solely dependent on population size but also on demographic and economic factors.  


---
#### 8.2. Crime Resolution Rate by Province (2023)

Now, we will visualize the **percentage of crimes solved** in each province.  
This will show which areas have **higher or lower resolution rates**.  


In [10]:
dbGetQuery(con,
           "select province, sum(con_2023) as total_crime, 
		          sum(esc_2023) as total_solved,
                  round(cast(sum(esc_2023)/ sum(con_2023) * 100.0 as numeric), 2) as percent_crime_solved
		   from crime
		   group by province
		   order by percent_crime_solved desc;")

province               total_crime total_solved percent_crime_solved
1  Melilla                  4357        2743       62.96               
2  Ceuta                    3541        2021       57.07               
3  Ciudad Real             15698        8256       52.59               
4  Palencia                 4915        2557       52.02               
5  Palmas (Las)            56626       28717       50.71               
6  Cáceres                 11615        5816       50.07               
7  Asturias                33888       16528       48.77               
8  Almería                 33291       16078       48.30               
9  Lugo                     9155        4288       46.84               
10 Santa Cruz de Tenerife  47596       21955       46.13               
11 Granada                 41752       18806       45.04               
12 Zamora                   5477        2430       44.37               
13 Ourense                  9624        4261       44.27               
14 Pontevedra              33910       15002       44.24               
15 Huelva                  23463       10343       44.08               
16 Jaén                    21073        9262       43.95               
17 Cádiz                   59907       25926       43.28               
18 Rioja (La)              12152        5237       43.10               
19 Córdoba                 31810       13651       42.91               
20 Albacete                15618        6675       42.74               
21 Cantabria               24212       10331       42.67               
22 Cuenca                   7394        3138       42.44               
23 Málaga                  99025       41331       41.74               
24 Segovia                  5950        2483       41.73               
25 Soria                    3715        1548       41.67               
26 Badajoz                 23654        9834       41.57               
27 Alicante/Alacant       105373       43747       41.52               
28 Murcia                  74194       30514       41.13               
29 Toledo                  32325       13285       41.10               
30 Huesca                   7987        3264       40.87               
31 León                    16827        6814       40.49               
32 Ávila                    6235        2521       40.43               
33 Salamanca               12695        5095       40.13               
34 Zaragoza                38899       15545       39.96               
35 Castellón/Castelló      29150       11615       39.85               
36 Coruña (A)              42111       16640       39.51               
37 Baleares (Illes)        78509       31005       39.49               
38 Teruel                   3979        1565       39.33               
39 Burgos                  13970        5343       38.25               
40 Valladolid              20040        7570       37.77               
41 Valencia/València      152856       57090       37.35               
42 Guadalajara             12673        4428       34.94               
43 Lleida                  22354        7727       34.57               
44 Sevilla                102860       35366       34.38               
45 Navarra                 35094       11264       32.10               
46 Madrid                 411782      127310       30.92               
47 Tarragona               46897       14157       30.19               
48 Girona                  48223       14275       29.60               
49 Barcelona              392698       88457       22.53               
50 Araba/Álava             15676         176        1.12               
51 Bizkaia                 59227         485        0.82               
52 Gipuzkoa                32920         257        0.78

![Power BI Visualization](images/crime_solved.png)

**Álava, Bizkaia, and Gipuzkoa**  

Crime-solving rates in the Basque provinces Álava, Bizkaia, and Gipuzkoa appear significantly lower than in other regions. This may be due to differences in crime classification, law enforcement structures, or data reporting practices.
Unlike the rest of Spain, these provinces are policed by the Ertzaintza, the autonomous Basque police force, which may have different methodologies for recording and reporting case resolutions.
Further investigation is needed to understand whether these differences stem from variations in policing strategies, administrative processes, or other factors affecting data collection.

---
#### 8.3.  Crime Distribution by Type (2023)  

This visualization shows the **proportion of different crime types** in Spain.  
It highlights which offenses are most prevalent, providing insight into overall crime trends.  
  

In [13]:
dbGetQuery(con,
          "with type_crime_summ as (
           select crime_type, sum(con_2023) as sum_crime
           from crime 
           where con_2023 <> 0
           group by crime_type
           )
           select crime_type, 
                  sum_crime, 
                  round(cast(sum_crime * 100.0 / sum(sum_crime) over()as numeric), 2) as percentage
           FROM type_crime_summ
           order by sum_crime desc;")

crime_type                 sum_crime percentage
1  PATRIMONIO                 1837864   74.86     
2  CONTRA LAS PERSONAS         226574    9.23     
3  CONTRA LA LIBERTAD          158874    6.47     
4  SEGURIDAD COLECTIVA          77079    3.14     
5  ADMÓN. JUSTICIA              41883    1.71     
6  FALSEDADES                   40745    1.66     
7  ORDEN PÚBLICO                29983    1.22     
8  LIBERTAD SEXUAL              21744    0.89     
9  OTRAS INFRACCIONES PENALES   11703    0.48     
10 RELACIONES FAMILIARES         7554    0.31     
11 ADMÓN. PÚBLICA                 505    0.02     
12 LEGISLACIÓN ESPECIAL           464    0.02

![Power BI Visualization](images/crime_type.png)

**Key Observations**    

The data shows a significant concentration of crimes in Spain within a few categories. Property crimes ("Patrimonio") account for 74.86% of all reported incidents, making it the dominant category. Crimes against persons (9.23%) and crimes against freedom (6.47%) follow, while all other crime types combined represent less than 10% of total cases.  

**Possible Explanation**  
Spain is one of the most visited countries in the world, attracting millions of tourists each year. High tourism levels often correlate with an increase in petty crimes, particularly theft, fraud, and pickpocketing, which fall under the property crime category. Tourists, who may be less familiar with their surroundings, frequently become targets in busy urban areas and public transport hubs.  

This pattern suggests that crime prevention efforts should prioritize reducing property crimes, especially in high-tourism zones, as this category alone makes up the vast majority of reported incidents.  


---
#### 8.4. Crime Resolution by Type (2023)  

This visualization shows the distribution of solved crimes across different crime categories in Spain.  
It highlights which types of crimes make up the largest share of resolved cases, rather than the overall crime-solving rate.  

⚠ **Warning:** These percentages do **not** represent the probability of a crime being solved. Instead, they show the **proportion of each crime type among all solved cases**.  
This allows us to see which types of cases are more frequently pursued and resolved, providing insight into law enforcement priorities.  

🚨 **Important:** This graph should **not be compared directly to the previous one**.  
The previous visualization showed the total distribution of reported crimes, while this one focuses only on solved cases.  
The percentages here **do not indicate how many crimes are solved out of the total reported**—they simply show the breakdown of solved cases by crime type.  

In [14]:
dbGetQuery(con,
           "with type_crime_solv as (
            select crime_type, sum(esc_2023) as total_solved
            from crime 
            where esc_2023 <> 0
            group by crime_type
            )
            select crime_type, 
                   total_solved, 
                   round(cast(total_solved * 100.0 / sum(total_solved) over()as numeric), 2) as percentage
            from type_crime_solv
            order by total_solved desc;")

crime_type                 total_solved percentage
1  PATRIMONIO                 342640       40.56     
2  CONTRA LAS PERSONAS        195137       23.10     
3  CONTRA LA LIBERTAD         119929       14.20     
4  SEGURIDAD COLECTIVA         70301        8.32     
5  ADMÓN. JUSTICIA             40085        4.75     
6  ORDEN PÚBLICO               27959        3.31     
7  LIBERTAD SEXUAL             17017        2.01     
8  FALSEDADES                  15758        1.87     
9  OTRAS INFRACCIONES PENALES   8167        0.97     
10 RELACIONES FAMILIARES        6994        0.83     
11 LEGISLACIÓN ESPECIAL          392        0.05     
12 ADMÓN. PÚBLICA                353        0.04

![Power BI Visualization](images/solv_per.png)

**Crime Resolution by Type: Observation**   

This visualization does not reveal any significant anomalies.  
The distribution of solved crimes generally follows the pattern of reported crimes—categories with more total cases also have more resolved cases.  
The proportions are expected, as more frequent crime types naturally result in more solved cases within those categories.  


---
#### 8.5. Crime Trends and Resolution Rates (Last 10 Years)  

This visualization provides an overview of **total reported crimes** and **solved cases** in Spain over the past decade. 

**Note:** Before creating this visualization, I transformed the data in Power BI from a wide format to a long format to facilitate year-by-year analysis.

![Power BI Visualization](images/decada.png)

**Observation**

Over the past decade, the trends in total reported crimes and solved cases have followed a relatively consistent pattern. In years with higher crime rates, the number of solved cases also increased, and in years with lower crime rates, the number of solved cases decreased accordingly.  

One noticeable exception is the sharp decline in both total crimes and solved cases in 2020, which continued for two years. This is likely linked to pandemic-related restrictions, which reduced overall criminal activity.  

Another key observation is the gap between total crimes and solved cases. A possible explanation is the high proportion of property crimes ("Patrimonio"), which account for nearly three-quarters of all reported crimes. These include theft, fraud, and burglary—offenses that are inherently more difficult to investigate and solve compared to violent crimes or those with clear evidence.  



---
### Conclusion of the Project

Spain presents a diverse crime landscape, influenced by factors such as urbanization, tourism, and regional policing structures. The data reveals key trends and patterns that shape the country's crime dynamics:  

- **1. Urban vs. Rural Crime Rates**: 
As expected, crime rates are significantly higher in large metropolitan areas like Barcelona, Madrid, and the Balearic Islands, where tourism, population density, and economic activity contribute to more criminal opportunities. In contrast, provinces with lower crime rates, such as Lugo, Teruel, and Cáceres, tend to be more rural, less densely populated, and experience lower levels of transient movement.  

- **2. Property Crimes Dominate**  
Building on the first point, the most prevalent type of crime in Spain falls under "Patrimonio" (Property Crimes). This category includes petty theft, fraud, and similar offenses. The connection is clear: areas with higher human traffic naturally see more instances of pickpocketing, scams, and related crimes.

- **3. Gap Between Reported and Solved Crimes**  
A notable pattern in the data is the difference between registered and solved crimes. This is largely due to the nature of "Patrimonio" (Property Crimes), which account for 75% of all reported cases. These include theft, fraud, and burglary—offenses that often present fewer investigative leads, making them inherently more challenging to solve. This trend is consistent with global patterns, where property crimes generally have lower resolution rates compared to violent offenses.  


- **4. The Sharp Decline in Crime During 2020-2021**  
One of the most notable trends is the significant drop in crime rates in 2020 and 2021. This period coincided with the COVID-19 pandemic, which led to movement restrictions and economic slowdowns—factors that likely contributed to reduced criminal activity.

- **5. Understanding Crime Resolution Rates in the Basque Country**  
While some provinces report relatively high crime-solving rates (Melilla, Ciudad Real, and Palencia), Álava, Biscay, and Gipuzkoa—the three provinces forming the Basque Country—show notably lower resolution rates (around 1%). These provinces are policed by the Ertzaintza, the autonomous Basque police force, which may use different crime classification and reporting methodologies.

Potential factors influencing these figures could include variations in investigative approaches, data reporting practices, or jurisdictional policies. Further research would be needed to better understand these differences.

---

### Final Thoughts  

Crime is a part of any society, but understanding its patterns helps improve policies, adjust law enforcement strategies, and make cities safer for both residents and visitors.

Overall, crime trends in Spain remain relatively stable, though different provinces show significant variation in crime rates and resolution. Law enforcement efforts seem to adapt over time, but these regional differences suggest that a one-size-fits-all approach may not be the most effective. Each area may benefit from strategies and resources tailored to its unique crime situation.

At the end of the day, looking at crime data isn't just about seeing numbers—it’s about spotting trends, understanding why they happen, and figuring out how things can be improved. The more we analyze and learn, the better decisions can be made to create safer communities for everyone.
